App.js

```diff
@@ -1,76 +1,247 @@
 import "./styles.css";
+import React from "react";
 
-function MainContainer() {
-  return (
-    <div>
-      <colgroup className="MainContainer">
-        <SideBar />
-        <NewsFeedCol />
-        <TrendingCol />
-      </colgroup>
-    </div>
-  );
-}
+export default class FetchRandomUser extends React.Component {
+  state = {
+    loading: true,
+    people: []
+  };
 
-function SideBar() {
-  return (
-    <div className="NavBarCol">
-      <col />
-      <div classNAme="ButtonRow">
-        <button className="Icon1" />
-        <button className="Icon" />
-        <p>Home</p>
-        <button className="Icon" />
-        <p>Explore</p>
-        <button className="Icon" />
-        <p>Notifications</p>
-        <button className="Icon" />
-        <p>Messages</p>
-        <button className="Icon" />
-        <p>Bookmarks</p>
-        <button className="Icon" />
-        <p>Lists</p>
-        <button className="Icon" />
-        <p>Profile</p>
-        <button className="Icon" />
-        <p>More</p>
-      </div>
-      <button className="TweetButton">Tweet</button>
-    </div>
-  );
-}
+  async componentDidMount() {
+    const url = "https://api.randomuser.me/?results=50";
+    const response = await fetch(url);
+    const data = await response.json();
+    this.setState({ people: data.results, loading: false });
```

Comment:
It would be a good challenge to consider how you could support pagination (i.e. fetching for the next 50 users after the user scrolls to the bottom of the current feed.